src/compiler/types.ts

```diff
@@ -2722,7 +2722,7 @@ namespace ts {
 
     export interface ThrowStatement extends Statement {
         readonly kind: SyntaxKind.ThrowStatement;
-        readonly expression?: Expression;
+        readonly expression: Expression;

```

Comment:
Do we need to handle emitting an arbitrary non-empty expression for empty identifiers in the emitter for this? Or is it OK to just let the empty identifier get emitted?

Reply:
We already handle that. There was one test difference showing the change, as we went from `throw;` to `throw ;`. 

I think this is better to do now because one of these days I want to propose `throw;` to TC39 as a means to rethrow an exception in a `catch {}` block (without a reachable binding), in which case having a missing expression would have a different meaning...